gapic/src/main/com/google/gapid/perfetto/models/SliceTrack.java

```diff
@@ -67,8 +67,48 @@ protected Slice buildSlice(Row row, ArgSet args) {
 
   public static SliceTrack forGpuQueue(QueryEngine qe, GpuInfo.Queue queue) {
     return new WithQueryEngine(qe, "gpu_slice", queue.trackId) {
+      private final String GPU_COLUMNS = "render_target, render_target_name, render_pass, render_pass_name, command_buffer, command_buffer_name";
```

Comment:
We can do this abstraction, I think, in a cleaner way, not requiring `ArgSet` to be mutable. After addressing some of the other comments, I can work with you to do it.

Reply:
I've actually cleaned it up myself and removed the changes from ArgSet. Please take a look now.

Reply:
Please add a TODO here to remove the copy-pasted code.